lib/common/injector.dart

```diff
@@ -163,20 +168,9 @@ Future<void> setup() async {
   final authenticatedDio = Dio(); // Authenticated dio instance for AU servers
   authenticatedDio.interceptors.add(AutonomyAuthInterceptor());
   authenticatedDio.interceptors.add(LoggingInterceptor());
+  authenticatedDio.interceptors.add(ConnectingExceptionInterceptor());
   (authenticatedDio.transformer as SyncTransformer).jsonDecodeCallback =
       parseJson;
-  dio.interceptors.add(RetryInterceptor(
```

Comment:
please add a another new dio.

Reply:
i remove that because that dio already add RetryInterceptor in dio_util